option_test.go

```diff
@@ -58,3 +58,29 @@ func TestTimeFormat(t *testing.T) {
 		t.Fatalf("diff = %q, want %q", got, want)
 	}
 }
+
+func TestZeroFields(t *testing.T) {
+	type T struct{ A, B int }
```

Comment:
Nit: rename this type? There are currently three different `T`s floating around. (This one, testing.T, and the type param in ZeroFields.)

Reply:
I had the same feeling. I'll change to C for "case" unless you have something in mind.

Reply:
updated.